inject.js

```diff
@@ -95,8 +95,8 @@ chrome.runtime.sendMessage({}, function(response) {
     tc.videoController.prototype.initializeControls = function() {
       var document = this.document;
       var speed = parseFloat(tc.settings.speed).toFixed(2),
-        top = Math.max(this.video.offsetTop, 0) + "px",
-        left = Math.max(this.video.offsetLeft, 0) + "px";
```

Comment:
Hmm, why did we need to drop the offset's? 

Is the fix here the special case in the switch below, or the code above?

Reply:
They are causing location bug and it's useless if I'm correct

Reply:
switch below: fix for unclickable overlay
code above(offset): fix for location of overlay